Updates/0333_dbscript_string_broadcast_text_id.sql

```diff
@@ -0,0 +1,200 @@
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000215	=> "Apothecary Zamah smiles."
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000150	=> "Well, I am off than. Take care, Harlan!"
+# oo mal [263] "Well, I'm off then. Take care, Harlan."
+
+UPDATE dbscript_string SET broadcast_text_id = 263 WHERE entry = 2000000150;
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000054	=> "Well, ok. I am now sending people . We need find and help him."
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000050	=> "Hello Grimclaw"
+# TOO MUCH
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000051	=> "What happens?"
+# TOO MUCH
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000052	=> "Volcor was gone?"
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000053	=> "You know where he is?"
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000030	=> "This makes sense!"
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000024	=> "And even more!"
+# TOO MUCH
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000023	=> "Look out! More are coming!"
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000022	=> "Look! Minions of Terokk are approaching!"
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000021	=> "I need to consult Noram and Horatio."
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000020	=> "That's interesting..."
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000186	=> "An abomination of the undead is approaching!"
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab 2000005545	=> "Thank you. I thought I was going to die."
+# ++ mal [12222] "Th... thank you. I thought I was going to die."
+
+UPDATE dbscript_string SET broadcast_text_id = 12222 WHERE entry = 2000005545;
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000166	=> "Sarath fades into the mist..."
+# FAR FOUND
+# oo mal [1476] "%s fades into the mist..."
+# oo mal [72076] "The ghost fades away into the mist..."
+
+# UPDATE dbscript_string SET broadcast_text_id = XXXXX WHERE entry = 2000000166;
+
```

Comment:
I think bid=1476 is the right one. And as a side note, "Sarath" is wrong, he's called Sargath.